test/extensions/filters/http/fault/config_test.cc

```diff
@@ -46,6 +46,29 @@ TEST(FaultFilterConfigTest, FaultFilterCorrectJson) {
   cb(filter_callback);
 }
 
+TEST(FaultFilterConfigTest, SimpleConfigServerContext) {
```

Comment:
nit: make call this test `FaultFilterCorrectJsonWithServerContext`, which matches the name of other test.